packages/angular/src/generators/ngrx/lib/validate-options.ts

```diff
@@ -19,8 +22,14 @@ export function validateOptions(
   }
 
   const angularVersion = getInstalledAngularVersion(tree);
+
+  const installedNgRxVersion = getInstalledPackageVersionInfo(
+    tree,
+    '@ngrx/store'
+  );
+
   const parentPath = options.parent ?? options.module;
-  if (parentPath && lt(angularVersion, '14.1.0')) {
+  if (parentPath) {
```

Comment:
nit: the check for `parentPath` is not needed. It was also no needed in the previous code. The first thing we do in this function is to check either `module` or `parent` is provided, otherwise, we throw. So, we can't get here with a falsy `parentPath`.